youtube_dl/extractor/bandcamp.py

```diff
@@ -248,3 +253,71 @@ def _real_extract(self, url):
             'title': title,
             'entries': entries,
         }
+
+
+class BandcampWeeklyIE(InfoExtractor):
+    IE_NAME = 'Bandcamp:bandcamp_weekly'
+    _VALID_URL = r'https?://(?:www\.)?bandcamp\.com/?\?(?:[^/]*&)?show=(?P<id>\d+)(?:$|[&#])'
+    _TESTS = [{
+        'url': 'https://bandcamp.com/?show=224',
+        'md5': 'b00df799c733cf7e0c567ed187dea0fd',
+        'info_dict': {
+            'id': '224',
+            'ext': 'opus',
+            'title': 'BC Weekly April 4th 2017: Magic Moments',
+            'description': 'Stones Throw\'s Vex Ruffin, plus up and coming singer Salami Rose Joe Louis, in conversation about their fantastic DIY albums.',
+        }
+    }, {
+        'url': 'https://bandcamp.com/?blah@&show=228',
+        'md5': '894ef5e83d7f653d99f644c5410be643',
+        'info_dict': {
+            'id': '228',
+            'ext': 'opus',
+            'title': 'BC Weekly May 2nd 2017: Shifting Gears',
+            'description': 'Featuring an interview with Malcolm Catto of The Heliocentrics, plus new music by Shabazz Palaces, Jane Weaver, and Morgan Guerin.',
+        }
```

Comment:
Yes, if it tests the same extraction scenario. There is only one extraction scenario in this code.